.github/workflows/central-publish.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       -   name: Set up JDK 11
           uses: actions/setup-java@v1
```

Comment:
Version of these actions have changed to v2, do a check

Reply:
fixed in a2e1304